### ISSUE TITLE
Tests open multiple times and end in infinite loop

### ISSUE THREAD

**Issue by jakubkarbowski** [electron/spectron#330]
I have a problem with creating tests for my application. Sometimes, when test starts after few seconds it is immediately restarted. This behaviour repeats ten times and test is closed with an error. 
This problem sometimes leads to failing test just, because of to long waiting time. 

![tests_loop](https://user-images.githubusercontent.com/22750083/46011231-85c26500-c0c5-11e8-9cd3-ec2198ed4390.PNG)

**Comment by daniloprates** [comment]
@jakubkarbowski did you find any solution for that? Facing the same issue.

**Comment by JimmyMac6996** [comment]
We are facing the same issue.

**Comment by hexaguin** [comment]
Yep, same issue here. Electron 5.0.6, with the standard structure for Electron Forge projects. Using the sample code from the README. Launching the test fills my screen with instances of my app. Interestingly enough, the actual `beforeEach` function is only getting called once, so I think it's something screwy to do with the actual launching of the app.

**Comment by hexaguin** [comment]
In my case, it turned out issue was caused by running the wrong version of Spectron. Downgrading to Spectron 7 fixed the issue for me.
